src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,57 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        while (robot.getX() != toX) {
```

Comment:
your solution is nice, because you make min moves when turning your robot, but it's harder to read code this way.
I suggest we work on readability and make code shorter ( you can always move your robot right or left until direction will be correct) That will cost us only 4 redundant moves in general, which is more than affordable and should improve code significantly

Reply:
Done